web/templates/left_sidebar.hbs

```diff
@@ -119,8 +119,8 @@
                 </a>
                 <span class="arrow sidebar-menu-icon starred-messages-sidebar-menu-icon"><i class="zulip-icon zulip-icon-more-vertical" aria-hidden="true"></i></span>
             </li>
-            <li class="tippy-left-sidebar-tooltip top_left_drafts top_left_row hidden-for-spectators" data-tooltip-template-id="drafts-tooltip-template">
-                <a href="#drafts" class="left-sidebar-navigation-label-container">
+            <li class="top_left_drafts top_left_row hidden-for-spectators">
+                <a href="#drafts" class="left-sidebar-navigation-label-container tippy-left-sidebar-tooltip" data-tooltip-template-id="drafts-tooltip-template">
```

Comment:
Is this one meant to be `tippy-views-tooltip`?

Reply:
Yeah, we should just use that regardless of if it is a view since we want them to behave the same.